src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+export enum STATUSES {
+  COMPLETED = 'completed',
+  ACTIVE = 'active',
+  ALL = 'all',
+}
```

Comment:
You don't need to write an enum name in upper case, check the example https://mate-academy.github.io/fe-program/js/extra/typescript
![image](https://github.com/user-attachments/assets/00957be0-0325-481a-b16d-036890c13d0f)